docs/io/unified.rst

```diff
@@ -230,6 +230,130 @@ At this time, the ``meta`` attribute of the :class:`~astropy.table.Table` class
 is simply an ordered dictionary and does not fully represent the structure of a
 FITS header (for example, keyword comments are dropped).
 
+Mixin Columns
+^^^^^^^^^^^^^^
+
+Starting with astropy 3.0 it is possible to store not only standard
```

Comment:
No need for "starting with astropy 3.0". If somebody wants to know when exactly a particular feature was implemented, they can check the changelog or the git history of the file.